File: part2/server/src/main/java/com/google/maps/gaming/zoinkies/models/EnergyData.java
package com.google.maps.gaming.zoinkies.models;

/**
 * A POJO class used by the Energy Station REST API.
 */
public class EnergyData {

  /**
   * Location id
   */
  private String Id;
  /**
   * Amount of energy restored for this player at this location.
   */
  private int AmountRestored;

  public String getId() {
    return Id;
  }

  public void setId(String id) {
    Id = id;
  }

  public int getAmountRestored() {
    return AmountRestored;
  }

  public void setAmountRestored(int amountRestored) {
    AmountRestored = amountRestored;
  }
}

File: part3/client/Zoinkies/Assets/Zoinkies/Scripts/Models/EnergyData.cs
namespace Google.Maps.Demos.Zoinkies {
  /// <summary>
  /// Model that provides information about restored energy.
  /// </summary>
  public class EnergyData {
    public string id{ get; set; }
    public int amountRestored{ get; set; }
    public override string ToString() {
      return "{Id: " + id +
             " AmountRestored: " + amountRestored +
             "}";
    }
  }
}
